### ISSUE TITLE
Import a new zone feature

### ISSUE THREAD

**Issue by skf82** [tuxis-ie/nsedit#111]
Seems this fnction stopped working after updating to 4.01 and nsedit from master branch.

It does not throw any error message, but simply just sets up a blank zone with just the nameservers and SOA. Unable to import any records.

**Comment by ruben-herold** [comment]
Here the same

**Comment by tuxis-ie** [comment]
Sorry, missen this one. Will look into this Monday

**Comment by skf82** [comment]
I have managed to go around the issue by deleting the NS records before zone import. Not selecting "overwrite nameservers" and just adding these manually after import

**Comment by tuxis-ie** [comment]
I can't reproduce this... Can you show me the zone and screenshots of when you are trying to do an import?

**Comment by skf82** [comment]
[test123.com.txt](https://github.com/tuxis-ie/nsedit/files/469753/test123.com.txt)

I have uploaded a sample zone i just created. If i import the zone with "overwrite nameservers" ticket it will just set up a blank zone.
If i import them with "overwrite nameservers" unticket. Then it works but i have to manually correct the nameservers to reflect the new ones

**Comment by skf82** [comment]
![snip1](https://cloud.githubusercontent.com/assets/16793382/18482484/8b45197e-79e0-11e6-9040-ed538558850d.PNG)
![snip2](https://cloud.githubusercontent.com/assets/16793382/18482485/8b4576d0-79e0-11e6-9dfe-86906af3f30a.PNG)

Here with screenshots

**Comment by bjoe2k4** [comment]
i can confirm this as well, however i don't see it as such big of a problem. In any case you have to type new nameservers, either before or after adding the zone. I would actually recommend to remove the entire feature to overwrite the nameservers.

**Comment by tuxis-ie** [comment]
I haven't had time to reproduce it yet, but I assume that one usually uses this function when you migrate a zone from other name servers to your, which are usually the same. So the function is very useful.

One thing, do your name servers have trailing dots?

**Comment by skf82** [comment]
Yes indeed the function is very useful, and i can sometimes do bulk imports of 20-30 zones at a time.
I do have trailing dots on the nameservers

**Comment by tuxis-ie** [comment]
Multiple zones at a time? How do you do that?

**Comment by skf82** [comment]
Apologies, it was badly worded. Not multiple zones at a time. But importing them one at a time
